include/deal.II/grid/tria_accessor.h

```diff
@@ -628,6 +628,12 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
    */
 
   /**
+   * Pointer to the @p ith vertex bounding this object. Throw an exception if
+   * <code>dim=1</code>.
+   */
+  TriaAccessor<0,dim,spacedim> vertex_accessor (const unsigned int i) const;
```

Comment:
That's what I tried first but I would have to add a new constructor in TriaRawIterator that can be used only for the vertices. On top of that, you cannot iterate on vertices so it's strange to return an iterator.